core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -456,8 +456,12 @@ public enum CoreError implements ScalarDbError {
       "Getting data neither in the read set nor the delete set is not allowed",
       "",
       ""),
-  CONSENSUS_COMMIT_READING_ALREADY_WRITTEN_DATA_NOT_ALLOWED(
-      Category.USER_ERROR, "0106", "Reading already-written data is not allowed", "", ""),
+  CONSENSUS_COMMIT_READING_ALREADY_WRITTEN_OR_DELETED_DATA_NOT_ALLOWED(
+      Category.USER_ERROR,
+      "0106",
+      "Reading already-written or already-deleted data is not allowed",
```

Comment:
```suggestion
      "Reading data already-written or already-deleted by the same transaction is not allowed",
```
The original message might sound like it includes cases where another transaction updates or deletes the record. What do you think?

Reply:
You are right. And I just realized we should say `Scanning` instead of `Reading` since we disallow only scanning already-written or already-deleted data.

```suggestion
      "Scanning data already-written or already-deleted by the same transaction is not allowed",
```

@josh-wong Could you please take a look at this from an English perspective?

Reply:
@josh-wong I’ve updated similar error messages in the same manner in 9ab7b475b39fd4001f494483ed6a14789413aa36. Could you please double-check them when you have a chance? 🙇 Apologies for the repeated request.

Reply:
No problem at all. The new changes LGTM!👍